src/View.js

```diff
@@ -70,7 +71,7 @@ export default class View extends React.Component {
   formatter = {
     type: ({ type }) => type?.label,
     status: ({ status }) => status?.label,
-    startDate: ({ startDate }) => (startDate ? <FormattedDate value={startDate} /> : ''),
+    startDate: license => (license.startDate ? <FormattedUTCDate value={license.startDate} /> : ''),
```

Comment:
Looks good. Can you please cleanup the unused `FormattedDate` import? And also 
```suggestion
    startDate: ({ startDate }) => (startDate ? <FormattedUTCDate value={startDate} /> : ''),
```